spec/lib/reporting_spec.rb

```diff
@@ -30,50 +26,41 @@
     user2.notes.create(body: 'test note body')
   end
 
-  let(:blank_note) do
-    user1.notes.create(body: '')
-  end
-
-  let(:nil_note) do
-    user1.notes.create(body: nil)
-  end
-
-  let(:whitespace_note) do
-    user1.notes.create(body: " \n \n \n")
-  end
-
   describe '.users' do
```

Comment:
The only tests we have at the moment are to do with notes. What's the best way to organise it so this is clear and make room for specs on the other values `Reporting` generates?